src/vivarium_public_health/results/disability.py

```diff
@@ -1,7 +1,7 @@
 """
-===================
-Disability Observer
-===================
+====================
+Disability Observers
```

Comment:
The risk docstring was plural so I wanted to be consistent. I prefer plural to leave room for future observers in each module